### PR TITLE
[CloudMinds]10318-RCU PM

### PR DESCRIPTION
**Pull request by Kenny-SZ**
This LWM2M Object includes RCU(Robotic Control Unit) PM information(counter and guage).

new pull request template

### CODE REVIEW

File: 10318.xml [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+        <Description><![CDATA[The Battery Level of the RCU.]]></Description>
+      </Item>
+      <Item ID="52">
+        <Name>Network Bandwidth</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>Mbps</Units>
```
Comment (seanmcilroy29) [inline 10318.xml]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10318.xml [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+        <Description><![CDATA[Wi-Fi Signal of the RCU.]]></Description>
+      </Item>
+      <Item ID="56">
+        <Name>UpLink Rate</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>Mbps</Units>
```
Comment (seanmcilroy29) [inline 10318.xml]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10318.xml [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+        <Description><![CDATA[UpLink Rate of the network used by the RCU.]]></Description>
+      </Item>	  
+      <Item ID="57">
+        <Name>DownLink Rate</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>Mbps</Units>
```
Comment (seanmcilroy29) [inline 10318.xml]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10318.xml [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+        <Description><![CDATA[Packet Loss Rate in percentage.]]></Description>
+      </Item>
+      <Item ID="59">
+        <Name>Network Latency</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>ms</Units>
```
Comment (seanmcilroy29) [inline 10318.xml:172]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10318.xml [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+      </Item>
+	  
+      <Item ID="300">
+        <Name>Battery Temperature</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Float</Type>
+        <RangeEnumeration/>
+        <Units>°C</Units>
```
Comment (seanmcilroy29) [inline 10318.xml]: Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

File: 10318.xml (hunk 1 above)
Comment (Kenny-SZ) [reply 10318.xml]: Have commented before.

File: 10318.xml (hunk 2 above)
Comment (Kenny-SZ) [reply 10318.xml]: Have commented before.

File: 10318.xml (hunk 3 above)
Comment (Kenny-SZ) [reply 10318.xml]: Have commented before.

File: 10318.xml (hunk 4 above)
Comment (Kenny-SZ) [reply 10318.xml:172]: The unit of time, millisecond.

File: 10318.xml (hunk 5 above)
Comment (Kenny-SZ) [reply 10318.xml]: Have commented before.

Comment (seanmcilroy29) [reply 10318.xml]: ``°C`` should be replaced with ``Cel``

**Comment by seanmcilroy29** [comment]
@Kenny-SZ - all the units submitted with ``°C`` should be replaced with ``Cel``. 
This is the correct unit with SenML

**Comment by Kenny-SZ** [comment]
Changed, thanks.

File: 10318.xml (hunk 1 above)
Comment (Kenny-SZ) [reply 10318.xml]: Mbps：megabit per second is a unit of data transfer rate.

File: 10318.xml (hunk 2 above)
Comment (Kenny-SZ) [reply 10318.xml]: Mbps：megabit per second is a unit of data transfer rate.

File: 10318.xml (hunk 3 above)
Comment (Kenny-SZ) [reply 10318.xml]: Mbps：megabit per second is a unit of data transfer rate.

File: 10318.xml (hunk 5 above)
Comment (Kenny-SZ) [reply 10318.xml]: Changed °C to Cel.

**Comment by Kenny-SZ** [comment]
Hi Sean, I have changed °C to Cel, but later on I found "°C" is still there , I don't know why.

I changed it again today.

**Comment by Kenny-SZ** [comment]
Changed % to /100.

**Comment by Kenny-SZ** [comment]
Changed Mbps to Mbit/s.

**Pull request merged**